lib/auth/helpers_mfa.go

```diff
@@ -92,72 +92,57 @@ func (d *TestDevice) Origin() string {
 	return d.origin
 }
 
-type authClient interface {
-	AddMFADevice(ctx context.Context) (proto.AuthService_AddMFADeviceClient, error)
+type authClientI interface {
```

Comment:
Why the rename?

Reply:
To avoid name clashes with variables named `authClient`.